app_server/test/services/UserService.js

```diff
@@ -85,6 +85,24 @@ lab.experiment('UserService tests', function () {
     });
   });
 
+  lab.test('getUserById valid arguments', function(done) {
+    UserService.createNewUser(bob).then(function (result) {
+      return UserService.getUserById(result.userId);
+    }).then(function(user) {
+      Code.expect(user.username).to.equal(bob.username);
+      Code.expect(user.password).to.equal(bob.password);
+      done();
+    });
+  });
+
+  lab.test('getUserById invalid arguments', function(done) {
```

Comment:
Should we have these tests both here and in `Service`?
As far as I can tell, it shouldn't improve either stability nor coverage as opposed to testing in `Service` only...?